warehouse/query-core/src/main/java/datawave/query/transformer/TermFrequencyQueryTransformer.java

```diff
@@ -1,27 +1,26 @@
 package datawave.query.transformer;
 
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Map.Entry;
-
+import datawave.marking.MarkingFunctions;
 import datawave.query.table.parser.TermFrequencyKeyValueFactory;
+import datawave.query.table.parser.TermFrequencyKeyValueFactory.TermFrequencyKeyValue;
+import datawave.webservice.query.Query;
+import datawave.webservice.query.exception.EmptyObjectException;
+import datawave.webservice.query.logic.BaseQueryLogicTransformer;
 import datawave.webservice.query.result.event.DefaultEvent;
 import datawave.webservice.query.result.event.DefaultField;
 import datawave.webservice.query.result.event.EventBase;
 import datawave.webservice.query.result.event.Metadata;
+import datawave.webservice.result.BaseQueryResponse;
 import datawave.webservice.result.DefaultEventQueryResponse;
 import org.apache.accumulo.core.data.Key;
 import org.apache.accumulo.core.data.Value;
 import org.apache.accumulo.core.security.Authorizations;
 import org.apache.commons.lang3.StringUtils;
 import org.apache.curator.shaded.com.google.common.collect.ImmutableList;
 
-import datawave.marking.MarkingFunctions;
-import datawave.query.table.parser.TermFrequencyKeyValueFactory.TermFrequencyKeyValue;
-import datawave.webservice.query.Query;
-import datawave.webservice.query.exception.EmptyObjectException;
-import datawave.webservice.query.logic.BaseQueryLogicTransformer;
-import datawave.webservice.result.BaseQueryResponse;
+import java.util.ArrayList;
```

Comment:
No changes required to this class

Reply:
Please see above

Reply:
My point on this one is that the ONLY changes you have to this class are import reordering.  Changes to this class are completely irrelevant to the pull request and can be reverted.